frontend/src/components/Login/InputButton.tsx

```diff
@@ -0,0 +1,14 @@
+import './InputButton.css';
+
+interface InputButtonProps {
+  name: string;
+  disable: boolean;
+}
+
+export function InputButton({ name, disable }: InputButtonProps) {
```

Comment:
This component is not needed. Replace the usage of this with a `button`. You do not need to add `type="submit"` to this button, because that is the default type.